pyproject.toml

```diff
@@ -30,11 +30,9 @@ filterwarnings = [
     # This is just to prevent pytest from exiting if pytest-xdist isn't installed.
     "ignore:Unknown config option.*looponfailroots:pytest.PytestConfigWarning",
 
-    # The following warning filters should be kept in sync with
-    # sentry.utils.pytest.sentry, and sentry.runner.settings.
-    # TODO(joshuarli): Address these as a prerequisite to testing on Django 2.1.
-    "ignore::django.utils.deprecation.RemovedInDjango20Warning",
+    # TODO(joshuarli): Enable this and address all after Django 2.0 lands.
```

Comment:
Why's the todo say enable? it's not commented out right now right

Reply:
Oh I guess I mean remove it, oh well I don't want to kick off CI again just to fix something that I'll basically be looking at soon, haha